### DISCUSSION TITLE
Routable Portals

### DISCUSSION THREAD

**Discussion by loxy** [taiga-family/taiga-ui#3010] [Q&A] [answered]
Hi guys,

I want to archive routable modals via Taiga UI portals, so that they open when reaching a specific URL. Are there any tools / helpers in Taiga UI? Or any idea or hint what could be a path to success?

Best regards,
Kersten

**Comment by splincode** [comment]
Great idea

**Reply by loxy** [reply, marked as answer]
Solution: 

- https://taiga-ui.dev/dialog/routable
- https://taiga-ui.dev/dialog/lazy-routable

See: https://github.com/Tinkoff/taiga-ui/pull/3369
